### PR TITLE
disable disconnect right after allocation.

### PR DESCRIPTION
**Pull request by studavekar**
Possible fix for https://github.com/ARMmbed/htrun/issues/170

### CODE REVIEW

**Comment by studavekar** [comment]
@bridadan @jupe 

Can you please review not sure if a disconnect was intentional.

File: mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def __remote_init(self):
 
         # Remote DUT connection, flashing and reset...
         try:
-            self.__remote_disconnect()
+            #self.__remote_disconnect()
```
Comment (jupe) [inline mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py]: it should be safe to remove whole line. it was originally workaround for one bug which is fixed long time ago already..

**Comment by theotherjimmy** [comment]
@studavekar This PR needs conflicts resolved (probably your other PR conflicted)

**Comment by studavekar** [comment]
@theotherjimmy conflicts are resolved.

**Pull request merged**